nix-support-production/lainchan.nix

```diff
@@ -50,6 +50,7 @@ in
     libiconv
     phpExtensions.memcached
     ghostscript
+    exiftool
```

Comment:
Same here, remove exiftool, libiconv, graphicsmagick, imagemagick, and gifsicle from systemPackages.